src/CVRF/cvrf_eval.c

```diff
@@ -74,6 +78,15 @@ void cvrf_model_eval_free(struct cvrf_model_eval *eval) {
 	oscap_free(eval);
 }
 
+void cvrf_get_os_info(const char *input_file, struct cvrf_model_eval *eval) {
+	struct oscap_source *source = oscap_source_new_from_file("/cpe/openscap-cpe-dict.xml");
```

Comment:
does this work? the hardcoded path doesn't seem right and we should definitely not hardcode any paths.